cdb2api/cdb2api.h

```diff
@@ -130,6 +130,7 @@ struct cdb2_effects_type {
     int num_updated;
     int num_deleted;
     int num_inserted;
+    unsigned long int cost;
```

Comment:
int64_t? long int is 32-bit in 32-bit builds.

Reply:
yep, thanks. was thinking to have uint64 but not sure if protobuf supports that.